common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -41,20 +45,28 @@ function (Sjson, AsyncProcess) {
                 languages = this.state.config.transcriptLanguages;
 
             this.loaded = false;
-            this.subtitlesEl = state.el.find('ol.subtitles');
             this.container = state.el.find('.lang');
-            this.hideSubtitlesEl = state.el.find('a.hide-subtitles');
+
+            this.hideCaptionsEl = state.el.find('a.hide-captions');
+            this.subtitlesEl = state.el.find('ol.subtitles');
```

Comment:
I recommend just targeting the class here, and leave off the elements. For example `.hide-subtitles` instead of `a.hide-subtitles`. This applies for any previous and future scenario here.